### PR TITLE
Finalized addition ofPangolin v1.1

### PR DESCRIPTION
**Pull request by DarianHole**
Add tool pangolin to galaxy

Anticipate many PR for this for the future as they release many new releases ahh

### CODE REVIEW

File: tools/pangolin/pangolin.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+<tool id="pangolon" name="Pangolin" version="@VERSION@+galaxy0">
```
Comment (bgruening) [inline tools/pangolin/pangolin.xml:1]: the ID seems to be wrong :(

File: tools/pangolin/pangolin.xml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                argument="--max-ambig"
+                />
+            <param name="min_length" type="integer" value="10000" min="0"
+                label="Minimum query length to attempt assignment"
+                argument="--min-length"
+                />
+        </section>
+    </inputs>
+    <outputs>
+        <data format="csv" from_work_dir="lineage_report.csv" name="lineage_report" label="${input1.name}_lineage_report" />
```
Comment (bgruening) [inline tools/pangolin/pangolin.xml:28]: can we output a "tabular" output here ... and convert the csv to tab as a second command after pangolin is called?

This way users do not need to convert it with a second tool and get all the tabular features of Galaxy.

File: tools/pangolin/pangolin.xml (hunk 1 above)
Comment (DarianHole) [reply tools/pangolin/pangolin.xml:1]: Oh noo

File: tools/pangolin/pangolin.xml (hunk 2 above)
Comment (DarianHole) [reply tools/pangolin/pangolin.xml:28]: Yes I will do that as well. I just have to specify tabular and it will auto convert correct

Comment (bgruening) [reply tools/pangolin/pangolin.xml:28]: No, this is just metadata. You need to do this as part of the tool I would recommend, e.g. with `sed 's/,/\t/g'`.

What you are suggesting is possible, but Galaxy would spin up a second job and this needs to be scheduled and so on ... so its more efficient if we do this in one job.

Comment (DarianHole) [reply tools/pangolin/pangolin.xml:28]: Sounds good, I'll add it that way! Thanks!

File: tools/pangolin/pangolin.xml (hunk 1 above)
Comment (DarianHole) [reply tools/pangolin/pangolin.xml:1]: Fixed in new PR

**Pull request merged**